syncer/checkpoint.go

```diff
@@ -616,7 +617,7 @@ func (cp *RemoteCheckPoint) createTable(tctx *tcontext.Context) error {
 			cp_table VARCHAR(128) NOT NULL,
 			binlog_name VARCHAR(128),
 			binlog_pos INT UNSIGNED,
-			binlog_gtid VARCHAR(256),
+			binlog_gtid TEXT,
```

Comment:
fix #867 (the real max length of VARCHAR relies on charset set, so simply use TEXT here).